stackslib/src/chainstate/stacks/auth.rs

```diff
@@ -314,6 +315,210 @@ impl MultisigSpendingCondition {
     }
 }
 
+impl StacksMessageCodec for OrderIndependentMultisigSpendingCondition {
+    fn consensus_serialize<W: Write>(&self, fd: &mut W) -> Result<(), codec_error> {
+        write_next(fd, &(self.hash_mode.clone() as u8))?;
+        write_next(fd, &self.signer)?;
+        write_next(fd, &self.nonce)?;
+        write_next(fd, &self.tx_fee)?;
+        write_next(fd, &self.fields)?;
+        write_next(fd, &self.signatures_required)?;
+        Ok(())
+    }
+
+    fn consensus_deserialize<R: Read>(
```

Comment:
Shouldn't this implement `DeserializeWithEpoch` and panic on `consensus_deserialize()`?

Reply:
It's fine without it as long as the block validation checks catch it.